thingif/src/main/java/com/kii/thingif/temporal/base/BaseAnd.java

```diff
@@ -0,0 +1,8 @@
+package com.kii.thingif.temporal.base;
+
+import java.util.List;
+
+public interface BaseAnd<T> {
```

Comment:
QueryClause is BaseClause
TriggerClause is BaseClause

We want to avoid concatenating QueryClause and TriggerClause don't we?